packages/react-menu/src/MenuDividerOnly.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Menu, MenuTrigger, MenuList, MenuItem, MenuDivider, MenuPopover } from './index';
+
+import { Button } from '@fluentui/react-button';
+import {
+  Cut20Regular as CutIcon,
+  ClipboardPaste20Regular as PasteIcon,
+  Edit20Regular as EditIcon,
+} from '@fluentui/react-icons';
+
+export const DividerOnly = () => (
```

Comment:
What does the **Only** mean in the story name?

Reply:
This means that the divider is for visual aesthetic only and does not have semantic HTML or accessible markup to delimit groups. Perhaps `VisualDividerOnly` is a better name ?

Reply:
went with `VisualDividerOnly`